src/arrayMethodSort.js

```diff
@@ -4,8 +4,30 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = defaultSort) {
+    let swapped;
+
+    do {
+      swapped = false;
+
+      for (let i = 0; i < this.length - 1; i++) {
+        if (compareFunction(this[i], this[i + 1]) > 0) {
+          [this[i], this[i + 1]] = [this[i + 1], this[i]];
+          swapped = true;
+        }
+      }
+    } while (swapped);
+
+    return this;
+  };
+
+  const defaultSort = (a, b) => {
+    // eslint-disable-next-line curly
+    if (String(a) < String(b)) return -1;
+    // eslint-disable-next-line curly
+    if (String(a) > String(b)) return 1;
```

Comment:
it's bad practice to use eslint-disable
```suggestion
    if (String(a) < String(b)) {
     return -1;
    };
    if (String(a) > String(b)) {
     return 1
    };
```